### ISSUE TITLE
Add support for include or import

### ISSUE THREAD

**Issue by pbennett** [tinymanorg/tealish#55]
I have many helper/utility functions that are shared between multiple related contracts.  Whether its shared constants, or common helper functions (like beaker/pyteal utils functions), Tealish currently forces everything to be in a single file.

This is forcing copy-paste duplication of large chunks of code in some cases.

A completely fleshed out import mechanism isn't particularly necessary, but a simple C like #include that injected from other files would address this problem well enough.

**Comment by fergalwalsh** [comment]
Yeah I see the need for this. It will require some work to make errors and sourcemaps work correctly so it's not trivial. Will definitely be coming though. 

In the meantime you might try a simple template system to generate the a final Tealish input file from a combination of other files and run that before the compile step.
